neurobooth_os/iout/metadator.py

```diff
@@ -82,9 +82,11 @@ def get_study_ids() -> List[str]:
 
 
 def get_subject_ids(conn: connection, first_name, last_name):
+    f_name = first_name.strip()
+    l_name = last_name.strip()
     table_subject = Table("subject", conn=conn)
     subject_df = table_subject.query(
-        where=f"LOWER(first_name_birth)=LOWER('{first_name}') AND LOWER(last_name_birth)=LOWER('{last_name}')"
+        where=f"LOWER(first_name_birth)=LOWER('{f_name}') AND LOWER(last_name_birth)=LOWER('{l_name}')"
```

Comment:
I'm not suggesting we fix at the moment given the use case, but looks like SQL injection potential. Could be the source of some special character issues.